### ISSUE TITLE
[BUG]: UGUI inputfiled can not operate in web，how to resolve this？

### ISSUE THREAD

**Issue by lijunhe1991** [Unity-Technologies/UnityRenderStreaming#897]
### Package version

3.1.0-exp.1

### Environment

```markdown
* OS:Windows10
* Unity version:2019.4.10f1
* Graphics API:
* Browser:Google Chorme
```

### Steps To Reproduce

* OS:Windows10
* Unity version:2019.4.10f1
* Graphics API:
* Browser:Google Chorme

**Comment by kannan-xiao4** [comment]
@lijunhe1991 
This issue is a duplicate of  https://github.com/Unity-Technologies/UnityRenderStreaming/issues/542 .
Our status does not change.

**Comment by lijunhe1991** [comment]
[BUG]: InputField, click down the Shift key will enter S, click down the CapsLock key will enter C, and click Backspace will enter B and cannot delete what has been entered #940
















在 2023-08-16 10:14:08，"Kazuki Matsumoto" ***@***.***> 写道：

Closed #897 as completed.
